### PR TITLE
update go format

### PR DESCRIPTION
**Pull request by pacoxu**
fix CI failure

### CODE REVIEW

File: kinder/pkg/cri/nodes/common/util.go [hunk 1]
```diff
@@ -215,7 +215,7 @@ func ContainerArgsForExternalEtcd(name string, args []string) []string {
 	return args
 }
 
-// TryUntil implements an helper that calls `try()`` in a loop until the deadline `until`
+// TryUntil implements an helper that calls `try()“ in a loop until the deadline `until`
```
Comment (SataQiu) [inline kinder/pkg/cri/nodes/common/util.go:218]:
```suggestion
// TryUntil implements an helper that calls `try()` in a loop until the deadline `until`
```

Comment (RA489) [reply kinder/pkg/cri/nodes/common/util.go:218]: nice catch @SataQiu

Comment (RA489) [reply kinder/pkg/cri/nodes/common/util.go:218]: nice catch @SataQiu

Comment (pacoxu) [reply kinder/pkg/cri/nodes/common/util.go]: Fixed.

Comment (RA489) [reply kinder/pkg/cri/nodes/common/util.go]: thanks

**Review by neolit123 — COMMENTED** [review]
/approve

**Comment by neolit123** [comment]
by the looks of things the new ginkgo cannot auto-ignore the -p flag when using dryrun (see the job spec for pull-kubeadm-kinder-upgrade-latest)  and we have and omit it.

somehow in kinder code / jobs we have to handle this.

> [1mOutput from proc 2:[0m
  Aug 16 05:02:05.701: INFO: The --provider flag is not set. Continuing as if --provider=skeleton had been used.
  I0816 05:02:05.701933   65630 e2e.go:116] Starting e2e run "965038c6-672c-4de0-a7f5-5971c567b904" on Ginkgo node 2
  [38;5;9mGinkgo detected configuration issues:[0m
  [1m[38;5;9mGinkgo only performs -dryRun in serial mode.[0m
    Please try running ginkgo -dryRun again, but without -p or -procs to ensure
    the suite is running in series.


or maybe there are more errors?

**Comment by neolit123** [comment]
@pacoxu would you have time to implement the above -p modification. perhaps we can check the incomming list of flags from the workflow and if -dryrun is there we can drop -p?

if you cannot i can check before friday.

**Comment by pacoxu** [comment]
> @pacoxu would you have time to implement the above -p modification. perhaps we can check the incomming list of flags from the workflow and if -dryrun is there we can drop -p?

I will check it next two days if I have time.

**Comment by pacoxu** [comment]
Firstly, I tried to use GINKGO_PARALLEL instead of `--parallel` of testing.
However, the task does not support `env`. Then I removed the `--parallel`

**Review by neolit123 — COMMENTED** [review]
/lgtm
/approve
/area kinder
/kind cleanup

i think this is ok, i did not see that we are adding --parallel explicitly

**Comment by neolit123** [comment]
thanks @pacoxu

**Pull request merged**